cli/Cargo.toml

```diff
@@ -61,7 +61,7 @@ once_cell = "1.12.0"
 deno_runtime = { version = "0.79.0" }
 deno_core = { version = "0.153.0" }
 deno_ast = { version = "0.19.0", features = ["transpiling"] }
-birdcage = { git = "https://github.com/phylum-dev/birdcage" }
+birdcage = { git = "https://github.com/phylum-dev/birdcage", branch = "andrea/abi-v2" }
```

Comment:
Would like to get this to main before merging.

Reply:
Just merged `birdcage` and pushed a commit with the branch spec removed.